ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -446,6 +458,33 @@ internal sealed class ArchiWebHandler : IDisposable {
 			return await UrlGetToHtmlDocumentWithSession(SteamCommunityURL, request).ConfigureAwait(false);
 		}
 
+		internal async Task<HashSet<ulong>> GetDigitalGiftCards() {
+			const string requestGifts = SteamStoreURL + "/gifts";
+			HtmlDocument response = (await WebLimitRequest(SteamStoreURL, async () => await WebBrowser.UrlGetToHtmlDocument(requestGifts).ConfigureAwait(false)).ConfigureAwait(false))?.Content;
+			if (response == null) {
+				Bot.ArchiLogger.LogGenericWarning(Strings.WarningFailed);
```

Comment:
This is expected and `WebBrowser` already prints a warning, we don't need second one.